src/components/MatchVideos/index.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles/index";
+
+const useStyles = makeStyles(theme => ({
+  embedContainer: {
+    position: "relative",
+    paddingBottom: "56.25%",
+    height: 0,
+    overflow: "hidden",
+    maxWidth: "100%",
+    marginBottom: theme.spacing(2),
+  },
+  embedIframe: {
+    position: "absolute",
+    top: 0,
+    left: 0,
+    width: "100%",
+    height: "100%",
+  },
+  button: {
+    marginRight: theme.spacing(1),
+  },
+}));
+
+const MatchVideos = ({ match, event }) => {
+  const classes = useStyles();
+  const youtubeSearchLink = `https://www.youtube.com/results?search_query=${encodeURIComponent(
+    `${match.getShortName()} ${event.year} ${event.name}`
```

Comment:
Use `getDisplayName(short=true)`

Reply:
It's shorter than that.
For example:
``getDisplayName`` => ``QM 1``
``getShortName`` => ``Q1``
This is what the site uses for the YouTube searching.

Reply:
Actually `getDisplayName(true)` => `Quals 1`, but yeah I see now that they are different.

That being said, I don't think there is a particular reason why that is why the old site uses this format for YouTube searching. I think it was designed before we had any clear naming scheme in mind. Using `getDisplayName()` (the long version) might actually increase searchability. Thoughts?

Reply:
Sounds good